### ISSUE TITLE
admin_key required for creating an organization is ignored.

### ISSUE THREAD

**Issue by msorens** [chef/automate#3295]
## Describe the bug
When one creates an organization attached to a chef server, the `admin_key` field is required but it is ignored.

## To Reproduce
Create a server:
```
$ curl -sSkH "api-token: $TOK" -X POST \
-d '{"name": "SERVER_NAME2", "description": "SERVER_DESCRIPTION2", "fqdn": "example.com", "ip_address": "127.0.0.1"}' \
https://a2-dev.test/api/v0/infra/servers?pretty
{
  "server": {
    "id": "d31e49ce-608b-46d4-b8b5-a819d19222ad",
    "name": "SERVER_NAME2",
    "description": "SERVER_DESCRIPTION2",
    "fqdn": "example.com",
    "ip_address": "127.0.0.1",
    "orgs_count": 0
  }
}
```
Grab the `id` value from above and plug in here to add an organization:
```
$ curl -sSkH "api-token: $TOK" -X POST \
-d '{"name": "ORG_NAME2", "admin_user": "ADMIN_USER", "admin_key": "KEY"}' \
https://a2-dev.test/api/v0/infra/servers/d31e49ce-608b-46d4-b8b5-a819d19222ad/orgs?pretty
{
  "org": {
    "id": "956c1773-d5ba-471b-8e0d-ceb37603e271",
    "name": "ORG_NAME2",
    "admin_user": "ADMIN_USER",
    "admin_key": "e2ac8d74-b419-42b3-b96c-aa2f738e8f18",
    "server_id": "d31e49ce-608b-46d4-b8b5-a819d19222ad",
    "projects": [
    ]
  }
}
```

Notice that the supplied `admin_key` value above is "KEY" and yet the object returns with "admin_key" filled in with a guid.

## Expected behavior
I am not sure what is supposed to happen, but a fix is needed either way.

### Scenario 1:

The `admin_key` should not be a user-specified field.

### Scenario 2:

The supplied value for the `admin_key` should be used.

**Comment by vsingh-msys** [comment]
As per the recent discussion, we are not allowing a user to edit the admin key during the update process, As the admin key is managed by `secrets-service` and currently working on https://github.com/chef/automate/pull/3285 somehow related to this issue. I have some query will discuss in Today's call.

**Comment by vsingh-msys** [comment]
Fixed in #3285

Current response:
```

curl -sSkH "api-token: $(get_admin_token)" -X POST -d '{"name": "SERVER_NAME2", "description": "SERVER_DESCRIPTION2", "fqdn": "example.com", "ip_address": "127.0.0.1"}' https://a2-dev.test/api/v0/infra/servers?pretty
{
  "server": {
    "id": "1bf66955-b7b0-40f1-afe6-2dd19f203a4f",
    "name": "SERVER_NAME2",
    "description": "SERVER_DESCRIPTION2",
    "fqdn": "example.com",
    "ip_address": "127.0.0.1",
    "orgs_count": 0
  }

```

Updated with earlier `admin_key` returned GUID of secrets-service with `credential_id`
```
curl -sSkH "api-token: $(get_admin_token)" -X POST \                               > -d '{"name": "ORG_NAME2", "admin_user": "ADMIN_USER", "admin_key": "KEY"}' \> https://a2-dev.test/api/v0/infra/servers/1bf66955-b7b0-40f1-afe6-2dd19f203a4f/orgs?pretty
{
  "org": {
    "id": "3f6d0955-cc9c-4248-8b69-e36c61814969",
    "name": "ORG_NAME2",
    "admin_user": "ADMIN_USER",
    "credential_id": "b75d7341-7d04-491d-880d-f09f7649179f",
    "server_id": "1bf66955-b7b0-40f1-afe6-2dd19f203a4f",
    "projects": [
    ]
  }
}
```
